src/exampleTypes/Date.js

```diff
@@ -56,7 +56,7 @@ let DateComponent = ({
   )
 
   return (
-    <div>
+    <div data-path={node.path}>
```

Comment:
These changes might be better generically as part of the contexturify HOCs we have so we don't have to remember to do this on every component